docs/quickstart.rst

```diff
@@ -0,0 +1,125 @@
+Quickstart
+==========
+
+Heudiconv Hello World: Using `heuristic.py`
+
+.. TODO convert to a datalad dataset 
+.. TODO ``datalad install https://osf.io/mqgzh/``
```

Comment:
unfortunately this specific install  would not work -- for datalad to install from osf that dataset should be uploaded there by datalad.   But we should be easily able to create a dataset with content coming from that zip on OSF if you know direct URL to that zip

here is what worked for me

```
datalad create -c text2git neuroimaging_core_sub-219
cd neuroimaging_core_sub-219
datalad download-url -m "Add original zip file content from https://osf.io/mqgzh" --archive https://osf.io/download/mqgzh/
```
and we can share it from datasets.datalad.org ... will we need anything else?

but I do not think that in this tutorial we should limit people to mandate to use datalad.  We should give them option to just do as was told -- download & extract

Reply:
and we should be able to point to that .zip as from this repo as well (in addition or even as the first choice to osf). This way we would be responsible to guarantee availability and have a fallback 

I am thinking to place such dataset under https://datasets.datalad.org/?dir=/repronim -- or could go under some new 'patterson/' under https://datasets.datalad.org/?dir=/labs . WDYT?

Reply:
+1 ﻿yarikoptic: Whats the procedure for getting datasets on datalad.org?  

> https://datasets.datalad.org/?dir=/repronim
> https://datasets.datalad.org/?dir=/labs

I'd vote "repronim" since I don't think Dianne will be maintaining it-- (IIUC her class data lives elsewhere now).

Reply:
could you create one on smaug or typhon following the recommendation above + may be annotate README on the source etc, and then point me to it